variable.c

```diff
@@ -189,15 +189,27 @@ is_constant_path(VALUE name)
  *    m.set_temporary_name(nil) # => #<Module:0x0000000102c68f38>
  *    m.name #=> nil
  *
- *    n = Module.new
- *    n.set_temporary_name("fake_name")
+ *    c = Class.new do
+ *       def foo
+ *       end
+ *    end
+ *    c.set_temporary_name("MyClass(with description)")
+ *
+ *    c.new # => #<MyClass(with description):0x0....>
+ *    c.instance_method(:foo) # => #<UnboundMethod: MyClass(with description)#foo() ...>
+ *    c.new.method(:foo) #<Method: MyClass(with description)#foo() ...>
```

Comment:
Missing `# =>` in this line

Reply:
@mame My point was to demonstrate that _all_ related objects are affected (as an emphasis on "why would this be useful").
Though we probably can just leave one of the two lines (`instance_method` vs `method`), did you mean that?